e2e/validation.go

```diff
@@ -12,13 +12,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-func ValidatePodRunning(ctx context.Context, s *Scenario) {
-	testPod := func() *corev1.Pod {
-		if s.VHD.OS == config.OSWindows {
-			return podHTTPServerWindows(s)
-		}
-		return podHTTPServerLinux(s)
```

Comment:
I dont see anymore `ValidatePodRunning` getting called with a linux pod.  was this function never used before ?

Reply:
It's a slightly different code path and is in scenario_helpers_test.go now:

![image](https://github.com/user-attachments/assets/f95811b4-ff5e-43ec-b016-3063f184986e)